extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -315,14 +345,36 @@ export class AmpAdUIHandler {
     return this.baseInstance_
       .attemptChangeSize(newHeight, newWidth, event)
       .then(
-        () => resizeInfo,
+        () => {
+          divertStickyAdTransition(this.baseInstance_.win);
+          if (
+            getExperimentBranch(
+              this.baseInstance_.win,
+              STICKY_AD_TRANSITION_EXP.id
+            ) === STICKY_AD_TRANSITION_EXP.experiment
+          ) {
+            this.setSize_(this.element_.querySelector('iframe'), height, width);
+          }
+          return resizeInfo;
```

Comment:
unused return?

Reply:
yep, I was looking at the following lines as a `.then` but its actually the catch, oops.